multi_out.cpp

```diff
@@ -0,0 +1,495 @@
+// Unified night image processor, producing keogram and startrails
+// Copyright 2021 Chris Kuethe <chris.kuethe@gmail.com>
+// Copyright 2018 Jarno Paananen <jarno.paananen@gmail.com>
+// Based on a script by Thomas Jacquin
+// Rotation added by Agustin Nunez @agnunez
+// SPDX-License-Identifier: MIT
+
+#include <getopt.h>
+#include <glob.h>
+#include <stdio.h>
+#include <sys/stat.h>
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <vector>
+#include "allsky.h"
+
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/opencv.hpp>
+
+struct config_t {
+  std::string img_src_dir, img_src_ext, dst_keogram, dst_startrails;
+  bool labels_enabled, keogram_enabled, startrails_enabled, parse_filename;
+  int img_width;
+  int img_height;
+  int fontFace;
+  int fontType;
+  int lineWidth;
+  int verbose;
+  union {
+    uint32_t textColor;
+    uint8_t a, r, g, b;
+  };
+  double fontScale;
+  double rotation_angle;
+  double brightness_limit;
+} config;
+
+// ---------------------------------------------------------------------------
+
+void parse_args(struct config_t*);
```

Comment:
Should this be:
`void parse_args(int argc, char** argv, struct config_t* cf);`

Reply:
Yes, yes it should.